core/commands/add.go

```diff
@@ -71,6 +71,30 @@ its filename. For example:
 You can now refer to the added file in a gateway, like so:
 
   /ipfs/QmaG4FuMqEBnQNn3C8XJ5bpW8kLs7zq2ZXgHptJHbKDDVx/example.jpg
+
+The chunker option, '-s', defines the chunking strategy for how 
+ipfs puts larger files into blocks. Blocks with same content can
+be deduplicated. The default is a fixed block size of
+256 * 1024 bytes, 'size-262144'. A variable block size can be
```

Comment:
Maybe instead of a variable block size, "Content defined chunking can be used in the form of the rabin chunker..."

I'm not super sure on how to best phrase this though. cc @lgierth @Stebalien

Reply:
First, for the `size-` chunker, I'd note that alternative sizes can be used (just for completeness).

As for wording, I'd use the second person: "Alternatively, you can use the rabin chunker for content defined chunking by specifying rabin-[min]-[avg]-[max] (where min/avg/max refer to the resulting chunk sizes)."

I don't think we really need to go into the respective merits here.